Logs/README.md

```diff
@@ -0,0 +1,13 @@
+# Logs
+```
+{
+  type: String,
+  timestamp: Time,
```

Comment:
Please make sure, the same time format used across RDS.

Reply:
Sure. Which model should I refer to?